stable/datacube-ows/values.yaml

```diff
@@ -59,8 +59,11 @@ owsConfig:
     repository: geoscienceaustralia/dea-datakube-config
     tag: latest
     pullPolicy: Always
+    cfg_folder: "/opt/dea-config/dev/services/wms/ows_refactored"
```

Comment:
Perhaps use a generic simplified config that is publicly accessible rather than the DEA specific one.

Reply:
The current default image is set to, this folder and is contained within the image, do you have a recommendation for a generic image we can use? or I can even create one?
```
owsConfig:
  image:
    registry: docker.io
    repository: geoscienceaustralia/dea-datakube-config
    tag: latest
    pullPolicy: Always
```

Reply:
We don't have simple one here either. I think a simple woudl be handy so folks no what its form is.

Reply:
changed the image to ows and the config to integration test config